express-hello-world/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const app = express();
+app.use(function (req, res, next) {
+  // console.log('Time:', Date.now());
```

Comment:
If an exercise requires you leave in a console log you can put this on the line above it so it will pass the commit checks:

```
// eslint-disable-next-line no-console
```